File: frontend/SportsVendor/src/shared/model/ShippingMethod.ts
export interface ShippingMethod {
  method: 'standard' | 'nextDay';
  name: string;
  price: number;
}

export const shippingMethods: ShippingMethod[] = [
  {
    method: "standard",
    name: 'Standard Delivery',
    price:  3.00
  },
  {
    method: "nextDay",
    name: 'Next Day Delivery',
    price:  8.00
  }
]

File: backend/OrderService/src/main/java/levin/schnabel/orderservice/structural/entity/ShippingMethod.java
package levin.schnabel.orderservice.structural.entity;

public enum ShippingMethod {

	STANDARD("standard", 3.00),
	NEXT_DAY("nextDay", 8.00);

	private final String name;
	private final double price;

	ShippingMethod(String name, double price) {
		this.name = name;
		this.price = price;
	}

	public String getName() {
		return name;
	}

	public double getPrice() {
		return price;
	}
}
